tests/TestRunner/tmp_cluster.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
Note: this file was carried over from master, where it is already written. I didn't write it or make any significant changes to it (except adding the `port` constructor arg)